lord-of-the-rings-library/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "lord-of-the-rings-library",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
main is the file that the user use npm to install to their own project. This means when the user uses `npm i` to install your library they need your index.js file. But you miss this fill in your project.

Reply:
You need the main file, which is  `index.js`. And in your `index.js` you should export all your code like:

```javascript
module.exports = {
kindOfCharacter:()=>{},
ifCharacterNotFound:()=>{}
....
}
```
or like a class: 
```javascript
module.exports = class className{
kindOfCharacter:()=>{},
ifCharacterNotFound:()=>{}
}
```

Reply:
I was trying to figure it out to make it work for RunKit! Thank you very much, this will definitely help, I'm making the changes now.